index.html

```diff
@@ -67,6 +121,57 @@
     // Submit the form
     $("#dlForm").submit();
   })
+
+  $("#plus").click(newDoc);
+  $("#minus").click(function() {
+    // TODO: replace this with something prettier
+    if (!confirm('OK to delete?')) {
+      return;
+    }
+    var n = $('.cur').next();
+    if (n.length === 0) {
+      n = $('.cur').prev();
+    }
+    $('.cur').remove();
+    delete localStorage[cur];
+    if (n.length>0) {
+      cur = n[0].id;
+      load(cur);
+    } else {
+      newDoc();
+    }
+  });
+
+  // Automatically run the transform after a keypress, throttled with a timer.
+  $("#in").keyup(function() {
+    if (state.timer) {
+      clearTimeout(state.timer);
+    }
+    state.timer = setTimeout(function(){
+      state.timer = null;
+      save();
```

Comment:
save runs the transform by calling `$("#go").click()`.  I'm going to make that a little more by refactoring a `go()` function.  I'll try your date approach, but I have a feeling that the timeout approach is going to feel more natural in practice, since with your approach, *every* time you start typing, you'll have a perf hiccup.  I think I have all the rest fixed; expect an update soon.